tools/common/Assembly.cs

```diff
@@ -384,6 +384,20 @@ public void ComputeLinkerFlags ()
 						if (Frameworks.Add ("OpenAL"))
 							Driver.Log (3, "Linking with the framework OpenAL because {0} is referenced by a module reference in {1}", file, FileName);
 						break;
+					case "PrintCore":
```

Comment:
Yeah it is. I added the previous two since they exist in both XM and XI and I did not want XI to get the behavior change. But I see the point. Moving.